code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1847,6 +1847,11 @@
 		species.anatomy_flags = rand(0,65535)
 	if(prob(5))
 		species.chem_flags = rand(0,65535)
+	if(prob(15))
+		species.tackleRange = max(0, rand(species.tackleRange-2, species.tackleRange+2))	//Leaving this with no upper limit is a choice I'm making today. God help us tomorrow.
+	if(prob(15))
+		species.tacklePower = max(0, rand(species.tacklePower*0.5, species.tacklePower*1.5))
```

Comment:
This is not equivalent; 0.5x is 1/2 while 1.5x is 1*1.5, or in other words this value on average will go down as the max impact of reduction is greater than the max impact of increase

Reply:
I don't understand. Am I retarded?

Reply:
Imagine that you have a number, 10. When you *0.5 it it becomes 5. When you *1.5 it it becomes 15. But if you *0.5 and then *1.5 you don't get the 10 back, you get 7.5, because the 1.5x is not equivalent to the 0.5x, 2x is equivalent to 0.5x, doubling and halving.